src/components/settings-row/settings-row.stories.mdx

```diff
@@ -44,6 +44,17 @@ All `children` are rendered in the input column to the right of the header colum
   </Story>
 </Canvas>
 
+### with HeadingType
+
+The `headingType` prop sets the HTML heading element of the `title` within the component which is rendered on the page. It can take values `"h1"` up to `"h5"`.
+<Canvas>
+  <Story name="headingType">
+    <SettingsRow headingType="h4" description="Description" title="This is a h4 Title">
```

Comment:
Since h4 is now the default we should show an example setting it to something different.